SI/Sem.04/Pract.04/Task06.cpp

```diff
@@ -0,0 +1,23 @@
+#include <iostream>
+
+bool checkSymbol(char S) {
+	if ((S >= 65 && S <= 90) || (S >= 97 && S <= 122)) {
```

Comment:
Re-write this function.
Think how to be able with a single return.
Can we remove the integer values for the letters?